services/app-api/src/postgres/contractAndRates/submitContract.ts

```diff
@@ -66,16 +64,16 @@ async function submitContract(
                 data: {
                     submitInfo: {
                         create: {
-                            updatedAt: groupTime,
+                            updatedAt: currentDateTime,
```

Comment:
Is the motivation here that we want to have the `updatedAt` at precisely the same timestamp across the transaction? I feel like even if we're inside a transaction the timestamp from just using the Prisma built in [updatedAt](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#updatedat) would only differ by milliseconds, and we might actually want to know when it actually persisted in the db. Display to users would still be the same date when made human readable.

Reply:
I think this was my change and I think we should revert it.  My change  wasn't based on any strong opinion or reason other than I was thinking they are all getting updated at the same tune --   I think we should go back to the db time if there's any reason we would use it which sounds like there is or could be @mojotalantikite